packages/bigtable/src/table.js

```diff
@@ -669,16 +687,27 @@ Table.prototype.insert = function(entries, callback) {
  *     deleted.
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error returned while making this request.
- * @param {object} callback.apiResponse - The full API response.
+ * @param {object[]} callback.statuses - A status for each entity transaction.
  *
  * @example
  * //-
  * // Insert entities. See {module:bigtable/table#insert}
  * //-
- * var callback = function(err, apiResponse) {
- *   if (!err) {
- *     // Mutations were successful.
+ * var callback = function(err, statuses) {
+ *   if (err) {
+ *     // Error handling omitted.
  *   }
+ *
+ *   // statuses = [
```

Comment:
I'm guessing this is MutateRows.  In the future, we plan on implementing a streaming response as an item is confirmed to be written into Bigtable.  For example, if there are 25 responses, and 5 complete really quick, we'll send a response for those 5 and keep on working on persisting the other 20.